pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -53,7 +52,7 @@ const (
 	kptFnLocalConfig  = "config.kubernetes.io/local-config"
 
 	kptDownloadLink = "https://googlecontainertools.github.io/kpt/installation/"
-	kptMinVersion   = "0.34.0"
+	kptMinVersion   = "0.38.1"
```

Comment:
Do you mind also adding a check to make sure the kptMaxVersion is less than 1.0.0 which does not have `kpt fn run`?